src/bidmanager.js

```diff
@@ -197,15 +198,20 @@ exports.addBidResponse = function (adUnitCode, bid) {
 
   // Video bids may fail if the cache is down, or there's trouble on the network.
   function tryAddVideoBid(bid) {
-    store([bid], function(error, cacheIds) {
-      if (error) {
-        utils.logWarn(`Failed to save to the video cache: ${error}. Video bid must be discarded.`);
-      } else {
-        bid.videoCacheKey = cacheIds[0].uuid;
-        addBidToAuction(bid);
-      }
+    if (config.getConfig('usePrebidCache')) {
+      store([bid], function(error, cacheIds) {
+        if (error) {
+          utils.logWarn(`Failed to save to the video cache: ${error}. Video bid must be discarded.`);
+        } else {
+          bid.videoCacheKey = cacheIds[0].uuid;
+          addBidToAuction(bid);
+        }
+        doCallbacksIfNeeded();
+      });
+    } else {
+      addBidToAuction(bid);
```

Comment:
Pretty sure, yes. Before the call to prebid-cache was added, video bids were still getting added to the auction. If the cache is disabled, we should still replicate that behavior.

Basically: If someone's _not_ trying to use the cache, then they must be doing their own thing with the video bid. We wouldn't want the cache behavior to prevent it from being added.

If someone _is_trying to use the cache, then they're _probably_ using `dfpAdServerVideo` too. In that case, a bid which didn't make it into the cache can only break things.